spark/src/test/scala/ai/chronon/spark/test/JoinTest.scala

```diff
@@ -61,7 +62,7 @@ class JoinTest {
   private val yearAgo = tableUtils.partitionSpec.minus(today, new Window(365, TimeUnit.DAYS))
   private val dayAndMonthBefore = tableUtils.partitionSpec.before(monthAgo)
 
-  private val namespace = "test_namespace_jointest"
+  private val namespace = "test_namespace_jointest" + "_" + Random.alphanumeric.take(6).mkString
```

Comment:
@pengyu-hou - I think there are a couple of places where this is hard-coded so we might need to update these as well , or pass the random string into it.

Reply:
https://sourcegraph.d.musta.ch/github.com/airbnb/chronon/-/blob/spark/src/test/scala/ai/chronon/spark/test/JoinTest.scala?L1128-1137

so maybe 
val random_suffix = Random.alphanumeric.take(6).mkString

and then use that in the references